cluster/gce/config-common.sh

```diff
@@ -162,4 +162,4 @@ export WINDOWS_INFRA_CONTAINER="gcr.io/gke-release/pause-win:1.2.1"
 # Storage Path for csi-proxy. csi-proxy only needs to be installed for Windows.
 export CSI_PROXY_STORAGE_PATH="https://storage.googleapis.com/gke-release/csi-proxy"
 # Version for csi-proxy
-export CSI_PROXY_VERSION="v0.1.0-gke.1"
\ No newline at end of file
+export CSI_PROXY_VERSION="v0.2.0-gke.0"
```

Comment:
nit: Add a newline at the end of the file. It's technically a POSIX requirement (and why GitHub puts the (-) symbol on the diff).

Reply:
approved so you can do this in a follow up if you want

Reply:
got it. thanks!